cardano-api/src/Cardano/Api/Fees.hs

```diff
@@ -922,8 +924,18 @@ makeTransactionBodyAutoBalance eraInMode systemstart history pparams
    era' :: CardanoEra era
    era' = cardanoEra
 
+   -- In the event of spending the exact amount of lovelace in
+   -- the specified input(s), this function excludes the change
+   -- output.
```

Comment:
Can we also comment that this does not save any fees. This is a subtle point so it's worth a proper comment explaining what is going on.